pkg/minikube/bootstrapper/certs.go

```diff
@@ -38,6 +38,11 @@ import (
 	"k8s.io/minikube/pkg/util"
 )
 
+const (
+	CACertificatesDir = "/usr/share/ca-certificates"
```

Comment:
is this only gonna work on linux ?

Reply:
The `openssl` command runs on Linux image so the host could be either Windows, Linux or Mac.